packages/modal/src/Modal.stories.js

```diff
@@ -96,6 +95,28 @@ export const WithModalHeader = () => {
   )
 }
 
+export const WithSmallModalHeaderAndWhiteBackgroundModalHeaderWithSetTextStyle = () => {
+  return (
+    <ModalStory
+      header={
+        <>
+          <SmallModalHeader />
+          <ModalHeader
+            bg='text.lightest'
```

Comment:
@pcln-james what is the alternative? I need support to make modal background white... think we need to keep bg?

Reply:
`color` is applied on `Flex` so if you set `background.lightest` that'll be white and the text should auto set to black because of the contrast logic in DS